README.md

```diff
@@ -1,37 +1,79 @@
-GADGETRON IMAGE RECONSTRUCTION FRAMEWORK
-=========================================
+# Gadgetron Image Reconstruction Framework
 
-Detailed installation instructions and manual is available at:
+## Setting up a development environment
 
-https://github.com/gadgetron/gadgetron/wiki
+The Gadgetron project uses [conda](ttps://conda.io) to manage dependencies. To set up a build environment, make sure you have [conda installed](https://docs.conda.io/projects/conda/en/latest/user-guide/install/index.html) and then:
 
-General Building Instructions (on Unix platforms) to install Gadgetron to system path
--------------------------------------------------
+```bash
+conda env create -f environment.yml
+```
+
+Using the `environment.yml` file provided in the root of this repository. There are a few dependencies that are not available from conda. Use the provided [script](docker/bootstrap-conda.sh) to install:
 
+```bash
+conda activate gadgetron
+./docker/bootstrap-conda.sh
 ```
-mkdir build
+
+## Building in conda environment
+
+In the conda environment (see above), you can build with:
+
+```bash
+mkdir -p build
 cd build
-cmake ../
-make
-sudo make install
+cmake -GNinja -DCMAKE_BUILD_TYPE=Release -DUSE_MKL=ON -DCMAKE_INSTALL_PREFIX=${CONDA_PREFIX} ../
```

Comment:
I am thinking we should hold off on defaulting to MKL since it may catch some people (that are building in different ways) by surprise. The others are mainly to be explicit about what we are doing.